### PR TITLE
Work on bug 513 xml validation fix and allowing empty value for a translation

### PR DESCRIPTION
**Pull request by PPAC37**
This adding to #530 modification, normaly allow #530 to pass CI test to allow empty value for translation in the xml validation (xsd shema with to hight restriction for the contente)  and in the load (TranslatorLanguage.java) of a language file.

### CODE REVIEW

File: src/test/resources/translator/language_no_dup_key.xsd [hunk 1]
```diff
@@ -21,7 +21,7 @@
                     <xs:complexType>
                         <xs:sequence>
                             <xs:element type="stringNotEmpty" name="key"/>
-                            <xs:element type="stringNotEmpty" name="value"/>
+                            <xs:element type="xs:string" name="value"/><!-- type from "stringNotEmpty" changed to "xs:string" to remove restriction to allow empty value -->
```
Comment (coliss86) [inline src/test/resources/translator/language_no_dup_key.xsd:24]: I don't think the comment is necessary, it just described what is in the code, not why.

**Comment by coliss86** [comment]
There 2 things in this PR : something about the center cartesian button which does nothing anymore and allowing empty value in the translation. 
The title of the PR (which will be the final commit) should reflect what is really done.

In my opinion, there should be 2 PR: one for the translation part, and a second on the home button.
One commit is on a subject only. Easy to say, hard to do, but it is case, easy :)
